CHANGELOG.md

```diff
@@ -48,11 +47,15 @@ Migrate to MUI 5, following the official [MUI Migration Guide](https://mui.com/g
     npx jscodeshift --extensions=ts,tsx --parser=tsx -t comet-admin/codemods/3.0.0/mui-style-engine.ts src/
     ```
 
-## @comet/admin-date-picker
+## @comet/admin-date-picker -> @comet/admin-date-time
+
+-   Added standalone `DatePicker` and `DateRangePicker` components for use outside of FinalForm
 
 ### Incompatible Changes
 
-The date-picker package has been removed.
+-   `@comet/admin-date-picker` has been renamed `@comet/admin-date-time`, and the date-picker components have been re-implemented from scratch.
+-   The props that previously were passed through to the [react-dates](https://github.com/react-dates/react-dates) components will no longer have any effect, as [react-date-range](https://github.com/hypeserver/react-date-range) is now used internally
```

Comment:
Do we need this information? Seems like an implementation detail

Reply:
I think the package rename & the fact that some props you could add to the old picker components will no longer work are important. 

But we could probably remove this part: `the date-picker components have been re-implemented from scratch`

Reply:
Maybe we could rephrase it to `The date picker library used internally has been changed from `[react-dates](https://github.com/react-dates/react-dates) to [react-date-range](https://github.com/hypeserver/react-date-range). Props specific to react-dates will no longer have any effect.

Reply:
> But we could probably remove this part: the date-picker components have been re-implemented from scratch

Yes, we should remove that.